_licenses/agpl-3.0.txt

```diff
@@ -4,7 +4,7 @@ nickname: GNU AGPLv3
 redirect_from: /licenses/agpl/
 source: http://www.gnu.org/licenses/agpl-3.0.txt
 
-description: The GNU GPL family of licenses is the most widely used free software license and has a strong copyleft requirement. When distributing derived works, the source code of the work must be made available under the same license. GNU AGPLv3 is distinguished from GNU GPLv3 in that hosted services using the code are considered distribution and trigger the copyleft requirements.
+description: Permissions of this strongest copyleft license are conditioned on making available complete source code of licensed works and modifications, which include larger works using a licensed work, under the same license when distributed. Copyright and license notices must be preserved. Contributors provide an express grant of patent rights to users. Additionally, when a modified version is used to provide a service over a network, the complete source code of the modified version must be made available to users of the service.
```

Comment:
I wouldn't assume the reader knows what "copyleft" is.

>  which include larger works using a licensed work

This reads a bit like legalese to me. There's traditionally been a conflict between completeness / thoroughness and readability. When possible to still be accurate, we've erred on the site of readability. The idea being, that the target user for the site is a first-or-second time project creator, who's just looking to license their project (e.g., coming from the dropdown on .com), and we should optimize for their 80% use case. Yes, there will be edges that won't be 100% accurate or fleshed out, but we're not trying to solve for every possible licensing concern, or write the canonical treatise on open source licensing. We're trying to help the user choose a license for their project.